keras/activations.py

```diff
@@ -34,6 +34,22 @@ def elu(x, alpha=1.0):
     return K.elu(x, alpha)
 
 
+def selu(x):
+    """Scaled Exponential Linear Unit. (Klambauer et al., 2017)
+
+    # Arguments
+        x: A tensor or variable to compute the activation function for.
+        alpha: A scalar, slope of positive section.
```

Comment:
Looks like alpha and scale are no longer arguments, so you can remove them from the docs here